src/Microsoft.Extensions.Logging.EventSource/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0-*",
+  "description": "A logger provider that sends data logged through ILogger interfaces to EventSource/EventListener infrastructure",
+  "packOptions": {
+    "tags": [
+      "EventSource",
+      "logging",
+      "ETW"
+    ],
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet/logging"
+    }
+  },
+  "buildOptions": {
+      "warningsAsErrors": true,
+      "keyFile": "../../tools/Key.snk",
+      "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.Logging.Abstractions": "1.0.0-*",
+    "System.Diagnostics.Tracing": "4.1.0-*",
+    "System.Threading": "4.0.11-*",
+    "Newtonsoft.Json": "8.0.3"
+  },
+  "frameworks": {
+    "net46": {
+      "dependencies": {
+        "System.Runtime": "4.0.0"
+      },
+      "imports": [ "dotnet" ]
+    },
+    "netcoreapp1.0": {
```

Comment:
This should be a `netstandard` TFM, which is for library. `netcoreapp` is only for apps. E.g. see this other logger: https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging.Debug/project.json#L25-L36.